Source/Configuration/Startup/TyphoonStartup.m

```diff
@@ -65,8 +66,30 @@ + (void)requireInitialFactory
     initialFactoryRequestCount += 1;
 }
 
++ (id<TyphoonDefinitionPostProcessor>)configPostProcessor
+{
+    NSString *fileName = [[NSBundle mainBundle] infoDictionary][@"TyphoonConfigFilename"];
```

Comment:
Better not to use [NSBundle mainBundle] here - there will be problems with unit tests.

Use - bundleForClass: instead.